gtsam/discrete/tests/testDecisionTree.cpp

```diff
@@ -20,13 +20,11 @@
 // #define DT_DEBUG_MEMORY
 // #define DT_NO_PRUNING
 #define DISABLE_DOT
-#include <gtsam/discrete/DecisionTree-inl.h>
-
+#include <CppUnitLite/TestHarness.h>
```

Comment:
Include order convention in gtsam is near -> far. It was already correct except all discrete includes should precede base includes.

Reply:
Auto-format oopsie